src/blueprint/from_config.rs

```diff
@@ -30,10 +30,15 @@ pub fn config_blueprint<'a>() -> TryFold<'a, ConfigSet, Blueprint, String> {
             .map(|upstream| blueprint.upstream(upstream))
     });
 
+    let links = TryFoldConfig::<Blueprint>::new(|config_set, blueprint| {
+        Valid::from(Links::try_from(config_set.links.clone())).map_to(blueprint)
+    });
```

Comment:
Used `Links::try_from` here to keep the standard, but `map_to(blueprint)` as we don't links property on blueprint.

Reply:
I fixed it.